docs/astrocut/index.rst

```diff
@@ -256,7 +256,62 @@ The cutout target pixel file format is decribed `here <file_formats.html#target-
                 1  PIXELS        1 BinTableHDU    222   144R x 12C   [D, E, J, 25J, 25E, 25E, 25E, 25E, J, E, E, 38A]   
                 2  APERTURE      1 ImageHDU        45   (5, 5)   float64  
 
-  
+You can also create cutout target pixel files out of TESS cube files stored on MAST's AWS open data bucket.
```

Comment:
Should we add a section header above this?

Reply:
I made it a subsection so that it still falls under the `TESS Full-Frame Image Cutouts` section. And the `Multithreading` subsection is just below it. Here's what it looks like now: 

<img width="732" alt="image" src="https://github.com/spacetelescope/astrocut/assets/32107699/80e1705f-0ae0-4868-84aa-68eddf6e569b">

Reply:
Can we call this multithreading instead? since multiprocessing and multithreading are very different things, people might get confused if we use the other term.

Reply:
Sure thing